test/unit/util/sku_token.test.js

```diff
@@ -19,6 +19,12 @@ test('sku token generation', (t) => {
         t.end();
     });
 
+    t.test('token only contains base-62 characters', (t) => {
+        const regex = /^[0-9a-zA-Z]+$/;
+        t.ok(regex.exec(skuToken));
```

Comment:
This test may not fail consistently if something was to break in the sku token generation. Is there a better way to test/verify that the base-62 character constraint is enforced?

Reply:
I don't think I understand what this test might not catch. We just want to check that the token consists only of characters 0-9, a-z, and A-Z right?

Reply:
If tokens are generated randomly, and there was a bug - only some tokens would fail this test and it would not be easy top replicate the test in any way to understand why it fails.